src/components/NavBar/NavBar.tsx

```diff
@@ -105,18 +103,18 @@ const NavBar: React.FunctionComponent = () => {
           Woohoo, youv&apos;e completed today&apos;s challenges!
         </Modal.Body>
         <Modal.Footer>
-          <Button variant="secondary" onClick={handleCloseCompleted}>
+          <Button className={styles.dcmodalBtn} onClick={handleCloseCompleted}>
             Nice!
           </Button>
```

Comment:
- What does this button does here other than closing the modal? Is `Nice` just like `cancel` button.
- `Nice` -> `Ok`
- Refer to [react-bootstrap](https://react-bootstrap.github.io/components/buttons/) and have an `variant` property for it. Make sure it is an `outline` propety